back/engines/commercial/analysis/spec/acceptance/taggings_spec.rb

```diff
@@ -68,4 +68,30 @@
       expect { Analysis::Tagging.find(id) }.to raise_error(ActiveRecord::RecordNotFound)
     end
   end
+
+  post 'web_api/v1/analyses/:analysis_id/taggings/bulk_create' do
+    with_options scope: :tagging do
+      parameter :tag_id, 'The ID of the tag to associate the input with', required: true
+      parameter :filters, 'The filters to select the inputs to associate the tag with', required: true
+    end
+    ValidationErrorHelper.new.error_fields(self, Analysis::Tagging)
+
+    let(:analysis) { create(:analysis) }
+    let(:analysis_id) { analysis.id }
+    let(:tag_id) { create(:tag, analysis: analysis).id }
+    let!(:inputs) { create_list(:idea, 3, project: analysis.project) }
+
+    example_request 'Bulk create taggings' do
```

Comment:
I got a little bit blocked on this test here - not fully sure why it's not working. Maybe we can have a look at it together if it's not immediately clear to you?

Reply:
I think it's failing because you are passing the filters in a `tagging` object in the request, but TaggingsController:40 is passing `params[:filters]` as if they're on the top level. You need `params[:tagging][:filter]`